.github/workflows/ci.yml

```
@@ -20,8 +20,11 @@ jobs:
           ruby-version: ${{ matrix.ruby-version }}
           bundler-cache: true
 
-      - name: Tests
-        run: bundle exec sus
+      - name: Isolated tests
+        run: bundle exec sus test/isolated/**/*rb
+
+      - name: Rails tests
+        run: bundle exec sus test/rails/**/*rb
```

I think this glob is missing a file extension separator.

```suggestion
      - name: Isolated tests
        run: bundle exec sus test/isolated/**/*.rb

      - name: Rails tests
        run: bundle exec sus test/rails/**/*.rb
```

---

I think the issue is `**/*` is interpreted differently in GitHub actions than it is on my local machine. Locally, it means “in this folder and/or sub folders”. In GitHub actions, it means “in sub folders”.

---

Perhaps this is a bash vs sh issue?

could try:

`/bin/bash -e "bundle exec sus test/isolated/**/*.rb"`

---

That doesn't work either. 😞